templates/server/index.html

```diff
@@ -1,460 +1,484 @@
+    <li class="p-list__item is-ticked">Ubuntu Server Live安装助手现在在联网的情况下自动更新到最新版本并获取最新安全漏洞补丁。提供自动安装的基础支持。</li>
+    <li class="p-list__item is-ticked">SSH支持2次身份认证（2FA）</li>
+    <li class="p-list__item is-ticked">WireGuard<sup>®</sup>——具有现代加密技术且简单易用的创新VPN技术</li>
+    <li class="p-list__item is-ticked">AppArmor3使系统更安全</li>
+    <li class="p-list__item is-ticked">更为灵魂的的bootloader，可容许故障磁盘启动。</li>
+    <li class="p-list__item is-ticked">在Microsoft Azure上对IPv6的支持更好</li>
+    <li class="p-list__item is-ticked">支持Amazon Web Services (AWS)最新的Instance Metadata Service（IMDSv2）</li>
+    <li class="p-list__item is-ticked">长期支持的Linux 5.4内核支持最新的硬件，且包含最新的安全更新。</li>
+    <li class="p-list__item is-ticked">软件版本更新：QEMU（v4.2）， libvirt（v6.0）， PHP（v7.4），Ruby（v2.7），GCC（V9.3），Python（v3.8），MySQL（v8.0），NGINX（v1.17）</li>
+  </ul>
+      <p><a class="p-link--external" href="https://wiki.ubuntu.com/FocalFossa/ReleaseNotes">更多内容请阅读发布公告</a></p>
     </div>
-  </div>
+</div>
 
+<div class="p-strip is-deep" style="padding-bottom: 2rem;">
+    <div class="u-fixed-width">
+      <h2 id="performance-and-versatility">性能和通用性</h2>
+        <p class="p-heading--three">为快速发展的企业提供灵活、安全、可随处部署的技术</p>
+    </div>
+</div>
 
-  <div class="p-strip--light is-deep is-bordered">
-    <div class="row">
-      <div class="col-12">
-        <h2>{{ releases.latest.short_version }} 新增功能</h2>
-        <ul class="p-list is-split">
-          <li class="p-list__item is-ticked">由Canonical提供9个月的支持</li>
-          <li class="p-list__item is-ticked">Linux 5.3内核</li>
-          <li class="p-list__item is-ticked">更新为qemu（v4.0），libvirt（v5.4），mysql（v8.0），postgresql（v11）等</li>
-          <li class="p-list__item is-ticked">新的修复程序集和对Ubuntu Server安装程序的刷新</li>
-          <li class="p-list__item is-ticked">新的Ubuntu优势体验</li>
-        </ul>
-        <p><a class="p-link--external" href="https://wiki.ubuntu.com/EoanErmine/ReleaseNotes">查看发布说明了解更多详情</a></p>
-      </div>
+<div class="row">
+    <div class="col-6">
+      <p>无论您要部署NoSQL数据库、Web服务场还是云，Ubuntu出色的性能和通用性都能满足您的需求。Ubuntu获得业内领先硬件OEM厂商的认证，并提供全面的部署工具，让您的基础架构可以物尽其用。</p>
     </div>
-  </div>
+    <div class="col-6">
+      <p>我们的定期发布周期意味着您可以使用最新，性能最高的开源软件。简化的初始安装和整合式的部署与应用程序建模技术，使Ubuntu Server成为简化大规模部署和管理的理想解决方案。</p>
+    </div>
+</div>
+
+<div class="p-strip is-deep is-bordered" style="padding-top: 2rem;">
+    <div class="u-fixed-width">
+      <h3 class="p-muted-heading u-align--center">可与您的硬件和软件一起使用</h3>
+      <ul class="p-inline-images">
+        <li class="p-inline-images__item">
+          <div class=" lazyloaded" data-noscript="">
+            <img src="https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,w_144,h_24/https://assets.ubuntu.com/v1/68713fd5-logo-lenovo.svg" srcset="https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,w_288,h_48/https://assets.ubuntu.com/v1/68713fd5-logo-lenovo.svg 2x" alt="Lenovo" width="144" height="24" loading="lazy" class="p-inline-images__logo">
```

Comment:
These images should all use the generator, too. You should be able to copy the code from here: https://github.com/canonical-web-and-design/ubuntu.com/blob/master/templates/server/index.html#L91